Rakefile

```diff
@@ -14,15 +14,20 @@ rescue LoadError
   end
 end
 
-begin
-  require "cookstyle/chefstyle"
+desc "Check Linting and code style."
+task :style do
```

Comment:
why aren't we using this in the Action?

Reply:
I have been adding Linting to both the Rakefile and as a task in .\github\workflows so that we can both test locally and as an automated action during a git push.